seed/views/datasets.py

```diff
@@ -36,48 +36,23 @@ class DatasetViewSet(viewsets.ViewSet):
     def list(self, request):
         """
         Retrieves all datasets for the user's organization.
-
```

Comment:
Lots of changes in this PR like this where I change the old docstring to a form that Swagger can interpret and render.